samples/samples/backup_sample.py

```diff
@@ -57,14 +58,17 @@ def create_backup(instance_id, database_id, backup_id):
 
 
 # [START spanner_restore_backup]
-def restore_database(instance_id, new_database_id, backup_id):
+def restore_database(instance_id, database_id, new_database_id, backup_id):
     """Restores a database from a backup."""
     spanner_client = spanner.Client()
     instance = spanner_client.instance(instance_id)
+    database = instance.database(database_id)
     # Create a backup on database_id.
 
     # Start restoring backup to a new database.
-    backup = instance.backup(backup_id)
+    expire_time = datetime.utcnow() + timedelta(days=14)
+    version_time = datetime.now()
+    backup = instance.backup(backup_id, database=database, expire_time=expire_time, version_time=version_time)
```

Comment:
`instance.backup()` is a factory for creating a Python object representing a Spanner backup. It does not create a backup. As such, passing these additional arguments does not do anything. The source database, expiry time and version time will all be based on the backend values which will be equal to the values passed when creating the backup.

Do you think updating the comment to `# Start restoring an existing backup to a new database.` would reduce confusion?

Reply:
Yep I've updated the comment :)